src/libraries/System.Net.Mail/src/System/Net/Mail/SmtpClient.Browser.cs

```diff
@@ -0,0 +1,160 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
You shouldn't need to add this file. We can keep the GeneratePlatformNotSupportedMessage and then add a file with the APIs we want to exclude from the not supported sources.

Reply:
Yes, but this was the only API that is excluded so it seemed easier to do this one file instead of creating a file that would need to be kept updated.  Thought it actually makes it easier to read and maintain.

If that is not the case we can of course go the direction you are suggesting here.

Reply:
Yeah that's true. Let's keep it this way.